lang/cs/Org.Apache.REEF.Client/Properties/Resources.resx

```diff
@@ -0,0 +1,133 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+  <!-- 
```

Comment:
It might make sense to replace this generic documentation with one specific to our use of resources here.